modules/default-values/assets/js/editor/commands/base.js

```diff
@@ -0,0 +1,60 @@
+import { pipe } from '../utils';
+
+export default class Base extends $e.modules.CommandBase {
+	/**
+	 * When finish to create or reset the default values this code will recreate the elements
+	 * on the document
+	 *
+	 * @param type
+	 * @param newSettings
+	 * @returns {Promise<void>}
+	 */
+	async recreateElements( type, newSettings = {} ) {
+		// Get all the elements that should recreate (e.g: type = 'heading' it will recreate all the heading)
+		const elementsToRecreate = this.getAllElementsForRecreate( type, newSettings );
+
+		// Fetch new widget config
+		await this.refreshWidgetsConfig( type );
+
+		// Will recreate all the elements with the same type to apply the default values.
+		$e.commandsInternal.run( 'document/elements/recreate', { models: elementsToRecreate } );
```

Comment:
Try use `document/elements/set-settings` for each container, it will also trigger render.